### PR TITLE
Rename back to kvm

### PR DESCRIPTION
**Pull request by anurse**
We aren't quite ready to finalize the name yet so we'll go back to kvm for now :)

/cc @muratg @Eilon @davidfowl @ChengTian

### CODE REVIEW

File: src/dotnetsdk.sh [hunk 1]
```diff
@@ -1,416 +0,0 @@
-# dotnetsdk.sh
```
Comment (anurse) [inline src/dotnetsdk.sh]: Guh. Another Rename+Edit turned into a Delete+Add :(

**Comment by Eilon** [comment]
I don't really know how to review this, so as far as I'm concerned, if all the tests pass, then check it in :smile:

**Comment by anurse** [comment]
Yeah... it's basically a big pile of renames...

**Comment by muratg** [comment]
Looks good.   :shipit:

**Pull request merged**